app/jobs/check_catalogues_job.rb

```diff
@@ -1,37 +1,34 @@
+require 'capybara'
+require 'capybara/poltergeist'
+require_relative '../../app/jobs/scrape_catalogues_job.rb'
+require_relative '../../app/jobs/delete_catalogues_job.rb'
+require_relative '../../app/jobs/capybara_quit_job.rb'

```

Comment:
Hi @eliotsykes , 

I've tested, and found that instead of using ```require_relative '../../app/jobs/check_catalogues_job.rb'```, I can use this instead ```require './app/jobs/check_catalogues_job.rb'```.

I will update this changes in the next PR fix.